src/api/submission/protocol_validations.py

```diff
@@ -243,9 +255,12 @@ class FishBeltProtocolValidation(TransectValidation):
     SAMPLE_UNIT = "fishbelt_transect"
 
     def validate(self):
-        results = [super(FishBeltProtocolValidation, self).validate()]
+        try:
```

Comment:
Would it be better/possible to put this in the `validate` method of both `TransectValidation` and `QuadratValidation`, and then these calls in individual protocol validations wouldn't have to change?:

```
        try:
            return [super().validate()]
        except SerializeValidationError:
            return ERROR
```